src/supertux/menu/sorted_contrib_menu.cpp

```diff
@@ -17,15 +18,17 @@
 #include "supertux/menu/sorted_contrib_menu.hpp"
 
 #include <sstream>
-#include "util/gettext.hpp"
+#include "boost/format.hpp"
+#include "gui/menu_manager.hpp"
+#include "gui/menu_item.hpp"
```

Comment:
Please separate system includes, dependency includes and SuperTux includes. You can find the details in [the coding style document](https://github.com/SuperTux/supertux/blob/master/CODINGSTYLE.md#includes).